tests/Kernel/CorporateBlocksFooterTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme\Kernel;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Class CorporateBlocksFooterTest.
+ */
+class CorporateBlocksFooterTest extends AbstractKernelTestBase {
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = [
+    'oe_corporate_blocks',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+    $this->installConfig(['oe_corporate_blocks']);
+  }
+
+  /**
+   * Test footer block rendering.
+   */
+  public function testFooterBlockRendering(): void {
+    // Setup and render footer block.
+    $config = [
+      'id' => 'oe_footer',
+      'label' => 'OpenEuropa footer block',
+      'provider' => 'oe_corporate_blocks',
+      'label_display' => '0',
+    ];
+
+    $render = $this->buildBlock('oe_footer', $config);
```

Comment:
We should override here the block footer configuration object by providing test configuration, so we can make sure that the template styled the footer correctly. I propose to have just 2 links per link category and test labels.